app/controllers/games_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def create
     game = Game.create(white_player_id: current_user.id)
+    current_user << game
```

Comment:
[You are right](http://api.rubyonrails.org/classes/ActiveRecord/Associations/CollectionProxy.html#method-i-3C-3C), your explicit version is actually the correct syntax.
Do ActiveRecord methods work in controllers? If not, I will use your second example.